content/docs/notebooks/why-use-jupyter-notebook.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Overview: Using Jupyter Notebooks in Kubeflow"
+description = "Why use Jupyter Notebooks in Kubeflow"
+weight = 5
++++
+
+
+The benefit of integrating Jupyter Notebooks service in Kubeflow is that:
+
+* In an enterprise environment, it integrates well with the rest of the infrastructure with respect to authentication and access control, and
```

Comment:
It looks like the entire topic discusses the benefits of using Kubeflow-hosted Jupyter notebooks in an enterprise environment. Unless you plan to add another section about benefits for an individual user, please remove the phrase "In an enterprise environment" from this first bullet point, and integrate that idea into your introductory sentence on line 8 ("The benefit of integrating...").

I've made another comment showing an example of how to do this.

Additionally:

- Please remove the word "and" from the end of this first bullet point. Write a full sentence as you do within your other bullet points.
- Do you mean `the rest of the Kubernetes infrastructure` or the broader enterprise infrastructure in general, beyond Kubernetes?